### DISCUSSION TITLE
Does it supports nested replies?

### DISCUSSION THREAD

**Discussion by iRajul** [Lakshan-Madushanka/laravel-comments#62] [Q&A] [answered]
I tried this plugin and found that only one level of nested reply is supported. or may be I am missing any configuration. Please confirm.

**Comment by Lakshan-Madushanka** [comment, marked as answer]
Nested replies aren't support.

**Comment by Lakshan-Madushanka** [comment]
yes from [3.0.0](https://github.com/Lakshan-Madushanka/commenter/releases/tag/3.0.0)

**Reply by iRajul** [reply]
Great, Thanks.
